lib/content-services/src/lib/content-node-share/services/shared-links-api.service.ts

```diff
@@ -21,6 +21,10 @@ import { Observable, from, of, Subject } from 'rxjs';
 import { AlfrescoApiService, UserPreferencesService } from '@alfresco/adf-core';
 import { catchError } from 'rxjs/operators';
 
+interface NodeOptions {
```

Comment:
Do you really need that interface? Why can't you reuse SharedLinkBodyCreate which is used by createSharedLink function?